holder.yml

```
@@ -196,7 +203,8 @@ paths:
        - oAuth2: []
        - zCap: []
       operationId: createPresentation
-      x-expectedCaller: "Holder Coordinator"
+      x-expectedCaller:
+        - Holder Coordinator
```

A workflow service could also conceivably use this endpoint.

```suggestion
        - Holder Coordinator
        - Workflow Service
```